extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -69,34 +69,30 @@ class AmpAccordion extends AMP.BaseElement {
           'Each section must have exactly two children. ' +
           'See https://github.com/ampproject/amphtml/blob/master/extensions/' +
           'amp-accordion/amp-accordion.md. Found in: %s', this.element);
-      this.mutateElement(() => {
```

Comment:
We did not initially have the mutateElement here before , Ali just added this a month ago (https://github.com/ampproject/amphtml/pull/9221) to fix an issue - https://github.com/ampproject/amphtml/issues/3586

This issue did not exist couple months before,I am not sure if we have fixed the real root cause of #3586